composer-psalm.json

```diff
@@ -25,7 +25,9 @@
     },
     "require": {
         "omnipay/common": "~2.0",
-        "vimeo/payment-gateway-logger": "^1.0"
+        "vimeo/payment-gateway-logger": "^1.0",
+        "composer/ca-bundle": "^1.1",
+        "ext-soap": "*"
```

Comment:
These are in composer.json but were missing here.

This wasn't causing any problems, since the psalm file is used for development only. But better to be consistent.